virttest/libvirt_vm.py

```diff
@@ -659,6 +659,32 @@ def add_memballoon(help_text, memballoon_model):
             logging.debug("vm.add_memballoon returning: %s", result)
             return result
 
+        def add_kernel(help_text, cmdline, kernel_path=None, initrd_path=None,
+                       kernel_args=None):
+            """
+            Adding Custom kernel option to boot.
+
+            : param help_text: string, virt-install help text
+            : param kernel_path: string, custom kernel path.
```

Comment:
doc string for cmdline can be added, also `: param kernel_path:` can be `:param kernel_path:` for all